src/solver/simulation/timeseries-numbers.cpp

```diff
@@ -722,14 +576,10 @@ void drawAndStoreTSnumbersForNOTintraModal(const array<bool, timeSeriesCount>& i
     for (auto& group: study.bindingConstraintsGroups)
     {
         const auto nbTimeSeries = group->numberOfTimeseries();
-        auto& value = group->timeseriesNumbers[0][year];
-        if (nbTimeSeries == 1)
-        {
-            value = 0;
```

Comment:
We don't need to assign **zero** to a TS number (in case associated size is **1**) : **zero** is the default value now